eng/common/pipelines/templates/steps/update-docsms-metadata.yml

```diff
@@ -32,6 +32,9 @@ parameters:
   - name: DocValidationImageId
     type: string
     default: ''
+  - name: AdditionalSteps
+    type: object
```

Comment:
I believe you can also use the `stepList` parameter type to get better schema validation when passing this in. https://docs.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops#parameter-data-types

Reply:
I will have a test.